scaleway/resource_k8s_cluster.go

```diff
@@ -229,6 +229,32 @@ func resourceScalewayK8SCluster() *schema.Resource {
 				Computed:    true,
 				Description: "The status of the cluster",
 			},
+			"private_network": {
```

Comment:
Is that needed rather than using private_network [datasource](https://registry.terraform.io/providers/scaleway/scaleway/latest/docs/data-sources/vpc_private_network) ?